src/index.html

```diff
@@ -12,6 +12,46 @@
     <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
 </head>
 <body>
-    
+    <h1 class="text-dark mx-5 my-4">Leaderboard</h1>
+    <main class="container gap-5 main d-md-flex flex-row">
+        <!-- score -->
+        <div class="score gap-3 d-flex flex-column">
+            <div class="head gap-3 d-flex flex-row">
+                <h3>Recent scores</h3>
+                <button type="button" class="btn btn-light shadow">Refresh</button>
+            </div>
+            
+            <table class="table table-striped">
+                <tbody class="table-body">
+                    <!-- <tr>
+                        <td>Ndikumana: 10</td>
+                    </tr>
+                    <tr>
+                        <td>Isaie: 19</td>
+                    </tr>
+                    <tr>
+                        <td>Eliane: 12</td>
+                    </tr> -->
```

Comment:
- [ ] Please, kindly remove this piece of old commented code in order to improve the cleanliness of the repo.

That way, we will meet the general best practices 💪